### PR TITLE
Add com.xnview.XnConvert (xnconvert-flatpak)

### PR DESCRIPTION
**Pull request by scx**
**XnConvert** is a powerful and free cross-platform batch image processing, allowing you to combine over 80 actions. Compatible with 500 formats. It uses the batch processing module of XnViewMP.
https://github.com/scx/flathub/tree/com.xnview.XnConvert

See also:
https://www.xnview.com/en/xnconvert/
https://github.com/scx/xnconvert-flatpak

### CODE REVIEW

**Comment by scx** [comment]
The vendor developer has been notified, but not answered yet.

**Comment by scx** [comment]
bot, build com.xnview.XnConvert

**Comment by scx** [comment]
bot, build com.xnview.XnConvert

**Comment by scx** [comment]
bot, build com.xnview.XnConvert

**Comment by pierrexn** [comment]
Thanks to make a Flatpak package

File: XnConvert.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+name: XnConvert
+buildsystem: simple
+sources:
+  - type: archive
+    url: https://download.xnview.com/old_versions/XnConvert-179-linux-x64.tgz
```
Comment (barthalion) [inline XnConvert.yaml]: It should use extra-data instead, as we're now indirectly violating their license by using test builds.

**Comment by eszlari** [comment]
@scx Of course it's your choice, but keep in mind, that splitting the manifest into multiple file makes updates through the Github UI more cumbersome (one commit for every file changed).

**Comment by bilelmoussaoui** [comment]
@eszlari unless you use a branch

File: XnConvert.yaml (hunk 1 above)
Comment (ramcq) [reply XnConvert.yaml]: Is this an old build, or just a stable URL to the current one? I don't see what in the license prohibits this, it's very clear:

```You are granted the right to use and to make an unlimited number of copies of this software.```

Comment (alexlarsson) [reply XnConvert.yaml]: Yeah, i think avoiding extra data when not needed due to licensing issues is a much better experience. We *do* have to be very careful about documenting licenses and stuff though, both in appdata and in the manifest.

Comment (scx) [reply XnConvert.yaml]: > Is this an old build, or just a stable URL to the current one?

There was an update on March 4. The current version is 1.80, but we have a URL for this as well:
https://download.xnview.com/old_versions/XnConvert-180-linux-x64.tgz

**Comment by scx** [comment]
@eszlari 

> @scx Of course it's your choice, but keep in mind, that splitting the manifest into multiple file makes updates through the Github UI more cumbersome (one commit for every file changed).

I share `ghostscript` and `ImageMagick` between **XnViewMP**, **XnConvert**, **XnSketch**, **XnRetro** as well as other projects, such as **Imagine**, **Kchmviewer**, **mtPaint**, **OCRFeeder**, **Qalculate**, **Unicode Plus**, **Vade Mecum Shelf**, **XaraLX**.

File: XnConvert.yaml (hunk 1 above)
Comment (scx) [reply XnConvert.yaml]: @pierrexn 

Can I assume you're okay with this?

**Comment by scx** [comment]
bot, build com.xnview.XnConvert

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.xnview.XnConvert

**Comment by scx** [comment]
@nedrichards 

Thanks!

**Pull request closed without merging**